.github/workflows/deploy.yaml

```
@@ -8,9 +8,34 @@ on:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    outputs:
+      should_run: ${{ steps.check_commit.outputs.should_run }}
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Get commit message
+        id: commit_message
+        run: echo "::set-output name=message::$(git log -1 --pretty=%B)"
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)